docs/functions.rst

```diff
@@ -1572,6 +1572,67 @@ Example:
         args:
           name: "test-snapshot-content-content-dfc8fa67-8b11-4fdf-bf94-928589c2eed8"
 
+.. _backupdatausingkopiaserver:
+
+BackupDataUsingKopiaServer
+--------------------------
+
+This function backs up data from a container into any object store
+supported by Kanister using Kopia Repository Server as Data-mover.
+
+.. note::
+   It is important that the application includes a ``kanister-tools``
+   sidecar container. This sidecar is necessary to run the
+   tools that capture path on a volume and store it on the object store.
+
+   Additionally, in order to use this function, a RepositoryServer CR is required
```

Comment:
We should point to the section on how to setup the kopia repository server setup.

Reply:
I think we don't have any documentation for that !, will add once the documentation is added